website/docs/r/fc_function.html.markdown

```diff
@@ -115,6 +115,7 @@ The following arguments are supported:
 * `runtime` - (Required) See [Runtimes][https://www.alibabacloud.com/help/doc-detail/52077.htm] for valid values.
 * `timeout` - (Optional) The amount of time your Function has to run in seconds.
 * `environment_variables` - (Optional, Available in 1.36.0+) A map that defines environment variables for the function.
+* `code_checksum` - (Optional, Available in 1.57.1+) The checksum (crc64) of the function code.
```

Comment:
* `code_checksum` - (Optional, Available in 1.57.1+) The checksum (crc64) of the function code. The value can be generated by data source [`alicloud_file_crc64_checksum`](https://www.terraform.io/docs/providers/alicloud/d/file_crc64.html)